jscomp/others/belt_MutableSetInt.mli

```diff
@@ -30,95 +30,605 @@
   and identity is not needed(using the built-in one)
 
   **See** [`Belt.MutableSet`]()
+
+  This module is [Belt.MutableSet](mutable-set) specialized with key type to be a `int` type.
+  It is more efficient in general, the API is the same with [Belt.MutableSet](mutable-set) except its key type is fixed, and identity is not needed (using the built-in one).
 *)
 
 
 # 38 "others/setm.cppo.mli"
 type value = int
-  
+(**
+  ```res prelude
+  type value = int
+  ```
+*)
+
 # 42 "others/setm.cppo.mli"
   (** The type of the set elements. *)
 
 
 type t
-(** The type of sets. *)
+(**
+  ```res prelude
+  type t
+  ```
+  The type of sets.
+*)
 
 val make: unit -> t
+(**
+  ```res sig
+  let make: unit => t
+  ```
+
+  Returns empty set.
+
+  ```res example
+  let set = Belt.MutableSet.Int.make()
+  ```
+*)
 
 val fromArray: value array -> t
+(**
+  ```res sig
+  let fromArray: array<value> => t
+  ```
+
+  Creates new set from array of elements.
+
+  ```res example
+  let s0 = Belt.MutableSet.Int.fromArray([1, 3, 2, 4])
+
+  s0->Belt.MutableSet.Int.toArray /* [1, 2, 3, 4] */
+  ```
+*)
+
 val fromSortedArrayUnsafe: value array -> t
+(**
+  ```res sig
+  let fromSortedArrayUnsafe: array<value> => t
+  ```
+
+  The same as [fromArray][#fromarray] except it is after assuming the input array is already sorted.
+*)
 
 val copy: t -> t
+(**
+  ```res sig
+  let copy: t => t
+  ```
+
+  Returns copy of a set.
+
+  ```res example
+  let s0 = Belt.MutableSet.Int.fromArray([1, 3, 2, 4])
+
+  let copied = s0->Belt.MutableSet.Int.copy
+  copied->Belt.MutableSet.Int.toArray /* [1, 2, 3, 4] */
+  ```
+*)
+
 val isEmpty: t -> bool
+(**
+  ```res sig
+  let isEmpty: t => bool
+  ```
+
+  Checks if set is empty.
+
+  ```res example
+  let empty = Belt.MutableSet.Int.fromArray([])
+  let notEmpty = Belt.MutableSet.Int.fromArray([1])
+
+  Belt.MutableSet.Int.isEmpty(empty) /* true */
+  Belt.MutableSet.Int.isEmpty(notEmpty) /* false */
+  ```
+*)
+
 val has: t -> value -> bool
+(**
+  ```res sig
+  let has: (t, value) => bool
+  ```
+
+  Checks if element exists in set.
+
+  ```res example
+  let set = Belt.MutableSet.Int.fromArray([1, 4, 2, 5])
+
+  set->Belt.MutableSet.Int.has(3) /* false */
+  set->Belt.MutableSet.Int.has(1) /* true */
+  ```
+*)
 
 val add: t -> value -> unit
+(**
+  ```res sig
+  let add: (t, value) => unit
+  ```
+
+  Adds element to set. If element existed in set, value is unchanged.
+
+  ```res example
+  let s0 = Belt.MutableSet.Int.make()
+  s0->Belt.MutableSet.Int.add(1)
+  s0->Belt.MutableSet.Int.add(2)
+  s0->Belt.MutableSet.Int.add(2)
+
+  s0->Belt.MutableSet.Int.toArray /* [1, 2] */
+  ```
+*)
+
 val addCheck: t -> value -> bool
+(**
+  ```
```

Comment:
I guess this was a `res sig`,... we can remove that as well. If there's no meaningful header, we can leave the doc header out for now

Reply:
Done